conf/reef/rados/3AZ-cluster.yaml

```diff
@@ -5,8 +5,6 @@ globals:
   - ceph-cluster:
       name: ceph
       node1:
-        networks:
```

Comment:
Doc string mentioned that the deploys 3 AZs in different subnets, but we are installing the cluster in the same subnet.

Reply:
Removed the subnets as most of the times, there aren't the free subnets mentioned in the conf.

These tests are capable of running on same subnet as well. removed the subnets as of now, so that we have test runs for the feature.